pimcore/lib/Pimcore/Log/Handler/ApplicationLoggerDb.php

```diff
@@ -62,9 +62,9 @@ public function write(array $record)
             'priority' => strtolower($record["level_name"]),
             'message' => $record["message"],
             'timestamp' => $record["datetime"]->format("Y-m-d H:i:s"),
-            'fileobject' => $record["context"]["fileObject"],
-            'relatedobject' => $record["context"]["relatedObject"],
-            'relatedobjecttype' => $record["context"]["relatedObjectType"],
+            'fileobject' => isset($record["context"]["fileObject"]) ? $record["context"]["fileObject"] : null,
```

Comment:
Since the isset checks also if the values is null (which in this context its a perfectly working statement) should we maybe replace "isset($record["context"]["fileObject"])" with "array_key_exists('context', $record) && array_key_exists('fileObject', $record['context'])" just to make it more future-proof, just in case php decides to interpret the isset as falsy instead of null